### ISSUE TITLE
[Question] Alarm widget filter

### ISSUE THREAD

**Issue by ganzuoni** [thingsboard/thingsboard#7720]
**Component**

 * UI

**Description**
Is there a way to specify a date range to filter alarm by clearTime in Alarm table?

**Environment**

 * OS:  Linux redhat 7
 * ThingsBoard: 3.2
 * Browser: any

**Comment by devaskim** [comment]
Timewindow to set date range + widget's *Alarm status list* set to `Cleared` ?

**Comment by ganzuoni** [comment]
No, it always filters on creation time

**Comment by Civilduino** [comment]
I don't think it is possible to filter the alarms by the timestamp of when they are cleared. Maybe you have to implement an action for that, see this issue #3919

**Comment by mykhailokornieiev** [comment]
I closed this ticket due to inactivity. If this question is still relevant to you, feel free to open a new ticket with a full description of your request.
